### PR TITLE
bump scalajs to 0.6.9

### PR DESCRIPTION
**Pull request by grzesiul**

### CODE REVIEW

**Comment by ghik** [comment]
ScalaJS is already 0.6.9

**Comment by grzesiul** [comment]
Indirectly, monifu 1.2 requires 0.6.8, so I didn't checked the latest version

**Comment by ghik** [comment]
Are these versions incompatible? I think you could just force version in SBT and be fine.

File: version.sbt [hunk 1]
```diff
@@ -1 +1 @@
-version in ThisBuild := "1.15.0"
\ No newline at end of file
+version in ThisBuild := "1.16.0"
```
Comment (ghik) [inline version.sbt]: If ScalaJS versions are compatible, this should be just `1.15.1`

**Pull request merged**